src/inspector_agent.cc

```diff
@@ -755,14 +755,17 @@ Agent::Agent(Environment* env)
 
 Agent::~Agent() {}
 
-bool Agent::Start(const std::string& path,
-                  const DebugOptions& options,
-                  std::shared_ptr<HostPort> host_port,
-                  bool is_main) {
+bool Agent::Start(
+    const std::string& path,
+    const DebugOptions& options,
+    std::shared_ptr<HostPort> host_port,
+    std::shared_ptr<std::vector<std::string>> allowed_http_get_hosts,
```

Comment:
Yes, it can be obtained from the DebugOptions.

But if we'd like to support changing the allow list at runtime from JavaScript API, it would be better to not directly accessing the one in DebugOptions.

Reply:
But changing it programmatically is not what's being asked. I'm not even sure that's a good idea, there might be security implications.